lib/node_modules/@stdlib/math/base/special/gamma-lanczos-sum/src/main.c

```diff
@@ -0,0 +1,74 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/gamma_lanczos_sum.h"
+#include "stdlib/constants/float64/pinf.h"
+
+/**
+* Calculates the Lanczos sum approximation.
+*
+* @param {number} x - input value
+* @returns {number} Lanczos sum approximation
```

Comment:
```suggestion
* @param x    input value
* @return     Lanczos sum approximation
```

We use separate markup for C implementations.